LoopKit/Persistence/Model.xcdatamodeld/Modelv3.xcdatamodel/contents

```diff
@@ -56,12 +56,12 @@
         <attribute name="hasLoopKitOrigin" attributeType="Boolean" usesScalarValueType="YES"/>
         <attribute name="programmedTempBasalRate" optional="YES" attributeType="Double" usesScalarValueType="NO" elementID="scheduledTempBasalRate"/>
         <attribute name="provenanceIdentifier" attributeType="String"/>
-        <attribute name="reason" attributeType="Integer 16" defaultValueString="0" usesScalarValueType="NO"/>
+        <attribute name="reason" attributeType="Integer 16" usesScalarValueType="NO"/>
```

Comment:
These changes will probably necessitate that the mapping files are updated.

Reply:
I will generate and update the mapping file in this PR.

Reply:
Mapping file updated, but am currently testing (it will take a while to build a Modelv2 app, generate data, update to latest Modelv3 app, test).

Reply:
Model mapping added and tested between v2 and latest v3.